src/auth-service/models/Subscription.js

```diff
@@ -63,9 +49,9 @@ SubscriptionSchema.methods = {
   toJSON() {
     return {
       _id: this._id,
-      email: this.subscribed,
-      isSystemUser: this.isSystemUser,
-      notifications: this.notifications,
+      email: this.email,
```

Comment:
Thanks @BenjaminSsempala , this is great progress. Awesome!

I am envisioning a future where we try our level best **NOT** to categorise users but rather, just approach it as AirQo users. In summary, the current status of things should not have a direct influence on our implementations. I am more than happy to sync further on the strategy.

Reply:
Thanks, @Baalmart We could have a sync to review the strategy. But to offer more light, the categories were more for the products, i.e., app notifications and analytics notifications.